test/test-format-validators.js

```diff
@@ -201,4 +201,67 @@ describe('format validators', function () {
       assert.ok(goodParamValue.valid);
     });
   });
+  describe('array', function () {
```

Comment:
I'm not sure these tests should be in `test/test-format-validators.js`.  This file is for testing the format validators, where `format` is used to further constrain the value of a `type`.  It would be better handled in https://github.com/apigee-127/sway/blob/master/test/test-parameter.js#L635